test/installation_node/install from source with thread parameter

```diff
@@ -25,8 +25,11 @@ nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION || "'nvm run $NVM_T
 # Remove the stuff we're clobbering.
 [ -e ../../$NVM_TEST_VERSION ] && rm -R ../../$NVM_TEST_VERSION
 
-# Install from source with 2 make jobs (and swapped arg order)
-nvm install -j 2 -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+# Speed up test for non-CI environment which may have more CPU threads
+[ -z "${CI-}"] && nvm_get_make_jobs
```

Comment:
It depends, it's more common to see 4~8 threads PC in the market, personally I'll use servers with more threads to do development works, actually I didn't really see a overhead for CI here, especially this part will finally be mocked in future, isn't it?